nodewatcher/modules/frontend/ip_space/static/css/tipsy.css

```diff
@@ -0,0 +1,125 @@
+/*Author of tipsy: https://github.com/jaz303, tipsy repository: https://github.com/jaz303/tipsy*/
```

Comment:
Spaces inside comments. So between `/*` and text. Feel free to use multi-line comment for a header.